src/routes/test.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import { db } from '../database';
+import { users } from '../database/schema.ts';
```

Comment:
Any chance these can just be relative imports without the `.ts` ending?

Reply:
I dont think so, it throws errors if i try "../database/schema";

Reply:
why does omitting the .ts matter?

Reply:
it does not matter